### DISCUSSION TITLE
How to add texture to greyscale colors for coBarplots and oncoplots?

### DISCUSSION THREAD

**Discussion by artgenko** [PoisonAlien/maftools#895] [Q&A]
Helo
I need to prepare figures in greyscale for the printed version of the publication is there any chance to add texture to oncoplots and coBarplots?
Best wishes
Artur

**Comment by PoisonAlien** [comment]
Hi,
I'm sorry for not getting back to you sooner. Could you please explain the grayscale and texture? Do you prefer to save the plot as non-colored?

**Reply by artgenko** [reply]
Hi, 
it is challenging to use only greyscale colors for oncoplot preparation, especially with clinical data description added as well, when you have to submit a manuscript for the journal and full-color figures need extra money. I wonder if it is possible to add some simple pattern of lines like in the links
https://stackoverflow.com/questions/2895319/how-to-add-texture-to-fill-colors-in-ggplot2
or
https://www.highcharts.com/forum/viewtopic.php?t=21890

to greyscale oncoplot ?
best wishes
A
